style.css

```diff
@@ -0,0 +1,404 @@
+  overflow: hidden;
+  position: relative;
+  margin: 1rem 2rem;
+}
+
+.arrival .box-container .box .image {
+  height: auto;
+  padding: 2rem;
+  text-align: center;
+  background: #fff;
+}
+
+.arrival .box-container .box .image img {
+  max-width: 100%;
+  height: auto;
+}
+
+.arrival .small-txt {
+  background-color: var(--bg-color);
+  color: var(--font-color);
+  font-size: 1rem;
+  padding: 5px 10px;
+  position: absolute;
+  left: 0;
+  bottom: 4.3rem;
+  width: 100%;
+  text-align: center;
+}
+
+.feature .row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: 2rem 0;
+  margin: 1rem auto;
+  width: 95%;
+}
+
+.feature .image-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-flow: column wrap;
+  padding: 2rem 0;
+  margin: 1rem auto;
+}
+
+.feature .banner-img {
+  max-width: 100%;
+  height: auto;
+  border: 0.1rem solid rgba(0, 0, 0, 0.3);
+  box-shadow: 0px 4px 50px rgba(0, 0, 0, 0.1);
+}
+
+.deal {
+  min-height: 60vh;
+}
+
+.deal .box-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.deal .box-container .box {
+  flex: 1 1 40rem;
+  height: 25rem;
+  margin: 2rem;
+  border-radius: 1rem;
+  overflow: hidden;
+  position: relative;
+  box-shadow: 0 0.3rem 0.5rem rgba(0, 0, 0, 0.3);
+}
+
+.deal .box-container .box img {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+}
+
+.deal .box-container .box .content {
+  position: absolute;
+  top: 55%;
+  left: 3rem;
+  transform: translateY(-50%);
+}
+
+.deal .box-container .box .content h3 {
+  font-size: 3.5rem;
+  color: #fff;
+  text-shadow: 0 0.3rem 0.5rem rgba(0, 0, 0, 0.3);
+}
+
+.deal .box-container .box .content p {
+  font-size: 2rem;
+  color: #fff;
+}
+
+.deal .box-container .box .content .btn {
+  margin: 1.4rem 0;
+}
+
+.form-container {
+  width: 60rem;
+  margin: 14rem auto;
+  padding: 3rem;
+  background-color: var(--bg-color);
+  border: 0.2rem solid var(--default-color);
+  box-shadow: 0px 4px 50px rgba(0, 0, 0, 0.08);
+  border-radius: 10px;
+  font-size: 2rem;
+}
+
+.form-container .title {
+  text-align: center;
+  padding-bottom: 2rem;
+  font-size: 5rem;
+  font-weight: 700;
+}
+
+.input input {
+  width: 100%;
+  padding: 0.5rem 1rem;
+  margin: 1rem 0;
+  border: 1px solid var(--default-color);
+  border-radius: 5px;
+  outline: none;
+}
+
+/* media queries  */
+
+@media (max-width: 991px) {
+  html {
+    font-size: 55%;
+  }
+
+  .header .header-1 {
+    padding: 2rem;
+  }
+
+  section {
+    padding: 3rem 2rem;
+  }
+}
+
+@media (max-width: 768px) {
+  html {
+    scroll-padding-top: 0;
+  }
+
+  body {
+    padding-bottom: 6rem;
+  }
+
+  .bottom-navbar {
+    display: block;
+  }
+
+  #search-btn {
+    display: inline-block;
+  }
+
+  .header .header-1 {
+    box-shadow: var(--box-shadow);
+    position: relative;
+  }
+
+  .header .header-1 .search-form {
+    position: absolute;
+    top: -115%;
```

Comment:
all non-negative values are valid percentage values above 100%. "-115%" values ,Is this correct?